### PR TITLE
HHA signal icons

### PR DESCRIPTION
**Pull request by Nakaner**
Björn Mahrt sent me some signals of HHA a few weeks ago. I have added them to the style.

### CODE REVIEW

File: styles/signals.mapcss [hunk 1]
```diff
@@ -1431,6 +1431,68 @@ node|z14-[railway=signal]["railway:signal:main"="AT-V2:trapeztafel"]["railway:si
...
+	icon-image: "icons/de-hha-h0-32.png";
+	icon-width: 6;
+	icon-height: 16;
+	text-allow-overlap: true;
+	allow-overlap: true;
+}
+
+/****************************************/
+/* DE Hamburger Hochbahn main signal    */
+/* without railway:signal:main:states=* */
```
Comment (rurseekatze) [inline styles/signals.mapcss]: I think it this comment line should be "with", not "without".

File: styles/signals.mapcss [hunk 2]
```diff
@@ -1431,6 +1431,68 @@ node|z14-[railway=signal]["railway:signal:main"="AT-V2:trapeztafel"]["railway:si
...
+	font-weight: bold;
+	icon-image: "icons/de-hha-h1-32.png";
+	icon-width: 6;
+	icon-height: 16;
+	text-allow-overlap: true;
+	allow-overlap: true;
+}
+
+/*******************************************/
+/* DE Hamburger Hochbahn distant signal    */
```
Comment (rurseekatze) [inline styles/signals.mapcss]: Remove 3 spaces between `signal` and `*/`.

**Comment by Nakaner** [comment]
@rurseekatze I fixed it. You can merge it.

**Comment by DerDakon** [comment]
You should also add railway:signal:direction check here.

**Comment by Nakaner** [comment]
@DerDakon I have fixed this.

File: styles/maxspeed.mapcss [hunk 3]
```diff
@@ -2373,3 +2373,57 @@ node|z14-[railway=signal]["railway:signal:speed_limit_distant"=lf6]["railway:sig
...
+}
+
+node|z14-[railway=signal]["railway:signal:speed_limit_distant"="DE-HHA:l1"]["railway:signal:speed_limit_distant:form"=sign]["railway:signal:speed_limit_distant:speed"=70]
+{
+	z-index: 430;
+	icon-image: "icons/de-hha-l1-70-sign-44.png";
+	icon-width: 22;
+	icon-height: 19;
+	allow-overlap: true;
+}
```
Comment (DerDakon) [inline styles/maxspeed.mapcss]: Missing newline.

**Comment by Nakaner** [comment]
@DerDakon wrote:
> Missing newline.

I have fixed this.

**Pull request merged**